# qsort_s gets us into trouble on Solaris 11
# It assumes the Bounds Checking Extension to
# C11, which we do not have. So just don't use it.
diff --git a/configure.ac b/configure.ac
index 826a688..ff2ce7e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1891,7 +1891,6 @@ AC_CHECK_FUNCS(ppoll)
 AC_CHECK_FUNCS(pread)
 AC_CHECK_FUNCS(pwrite)
 AC_CHECK_FUNCS(qsort_r)
-AC_CHECK_FUNCS(qsort_s)
 AC_CHECK_FUNCS(readlink)
 AC_CHECK_FUNCS(realpath)
 AC_CHECK_FUNCS(round)
